src/SpectodaRuntime.ts

```
@@ -0,0 +1,1657 @@
+
+              case Query.TYPE_SELECTED: {
+                {
+                  try {
+                    await this.connector.selected().then((result: any) => {
+                      item.resolve(result);
+                    });
+                  } catch (error) {
+                    item.reject(error);
+                  }
+                }
+                break;
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)